.github/ISSUE_TEMPLATE/mod_integration.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Mod integration 
+about: Suggest an integration for a mod
+labels: enhancement, "mod-integration"
+assignees: 'Nerdpie'
+
+---
+
+<!--
+## Before reporting
+ - Search for the suggestion here. It's possible someone's suggested it before!
+ - In particular, see issue #196 and the 'Mod integration' project.
+-->
+
+## Useful information to include:
+ - Explanation of how the feature/change should work.
+ - Some rationale/use case for a feature. I'd like to keep Plethora as minimal as possible, so I like to have a solid justification for each feature.
+ - Detailed use cases, possibly with pseudo-code, can improve the chances of your suggestion being implemented.
```

Comment:
Maybe "Detailed use cases, possibly with example code help make it easier to see the sort of things you'd like to interact with. This makes it much easier to implement your suggestion!"? I don't know, I'm not sure if it's any clearer.